### PR TITLE
[State transitions] Remove detect event

### PR DESCRIPTION
**Pull request by Nil20**
Farajaland PR: https://github.com/opencrvs/opencrvs-farajaland/pull/932

### CODE REVIEW

File: packages/workflow/src/features/registration/utils.test.ts [hunk 1]
```diff
@@ -38,7 +32,11 @@ describe('Verify utility functions', () => {
   })
 
   it('Generates proper birth tracking id successfully', async () => {
```
Comment (Zangetsu101) [inline packages/workflow/src/features/registration/utils.test.ts:34]:
```suggestion
  it('Generates proper birth tracking id successfully even if countryconfig returns 404', async () => {
```

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]
@Nil20 gateway is also probably using openhim url & a couple of other services most likely. The easiest way to look for them would be to search for the port "5001" in both core & farajaland

File: packages/workflow/src/features/registration/handler.test.ts [hunk 2]
```diff
@@ -25,6 +25,8 @@ import { cloneDeep } from 'lodash'
 import { BIRTH_BUNDLE, DEATH_BUNDLE } from '@opencrvs/commons/fixtures'
 import { server as mswServer } from '@test/setupServer'
 import { rest } from 'msw'
+import { TransactionResponse } from '@workflow/records/fhir'
+import { URLReference } from '@workflow/../../commons/build/dist/types'
```
Comment (Zangetsu101) [inline packages/workflow/src/features/registration/handler.test.ts:29]: import looks a bit weird

File: packages/user-mgnt/src/features/createUser/handler.test.ts [hunk 3]
```diff
@@ -242,30 +242,6 @@ describe('createUser handler', () => {
...
-    })
-
-    expect(fetch.mock.calls.length).toBe(3)
-    expect(fetch.mock.calls[2][0]).toEqual(
-      'http://localhost:5001/fhir/Practitioner/123'
-    )
-    expect(fetch.mock.calls[2][1].method).toEqual('DELETE')
-    expect(res.statusCode).toBe(500)
-  })
-
```
[the remark this answers is not in the document]
Comment (Nil20) [reply packages/user-mgnt/src/features/createUser/handler.test.ts:268]: My bad. Instead of updating the fhirUrl, removed the test.

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**